src/jquery/data.js

```diff
@@ -38,3 +39,42 @@ jQuery.data = function( elem, name, value ) {
 
 	return oldData.apply( this, arguments );
 };
+
+jQuery.fn.extend( {
+    data: function( key, value ) {
+        var args, result;
+        if ( arguments.length === 0 && typeof Proxy !== "undefined" ) {
+            result = oldFnData.call( this );
+	    if(!result) {
```

Comment:
`if` needs to be followed by the space. Please have a look at the GitHub Actions output, it shows you all the various ESLint errors of this PR.